docs/fsharp/style-guide/formatting.md

```diff
@@ -1092,16 +1092,24 @@ match l with
     | [] -> failwith "Couldn't find David"
 ```
 
-If the expression on the right of the pattern matching arrow is too large, move it to the following line, indented one step from the `match`/`|`.
+If one of the expressions on the right of the pattern matching arrow is too large, move it to the following line, indented one step from the `match`/`|`. Move all the other expressions to the next line as well for consistency.
```

Comment:
I don't interpret Don's last comment in the suggestion as setting this as the default, but rather an option.
I think the wording should reflect that (or the suggestion discussion continued about the default-ness of this)

Reply:
@dsyme set the label _style-guide-decided_, so I regard it as a marker that style guide is decided (final decision). As I understand, Don was not sure about tooling changes needed to support this style, but it should not affect the style and the recommendation itself.

Reply:
The label is fine.
What I do not see confirmed is that "defaultness" of this, I interpret the context of the suggestion that this should still remain a possible option only.

Reply:
Ok, looks like it can't be resolved without Don

Reply:
I agree with @T-Gro , here is what don said
> I agree with @isaacabraham that at option (4) listed above should be **an option**.

(emphasis mine)
cc: @vzarytovskii